Makefile

```diff
@@ -37,13 +37,37 @@ clobber:	TARGET = clobber
 setup:		TARGET = setup
 test:		TARGET = test
 component-hook:		TARGET = component-hook
+generate-package-kdl:	TARGET = generate-package-kdl
 
 .DEFAULT:	publish
 
-download setup prep build install pre-publish publish validate clean clobber \
-test component-hook unpack patch: $(SUBDIRS)
+download prep build install pre-publish publish validate clean clobber test component-hook generate-package-kdl unpack patch: $(SUBDIRS)
 
 $(SUBDIRS):	FORCE
 	@+echo "$(TARGET) $@" ; $(GMAKE) -s -C $@ $(TARGET)
 
 FORCE:
+
+# Bootstrap pkgdev before running component setup
+setup: bootstrap-pkgdev $(SUBDIRS)
+
+bootstrap-pkgdev:
+	@echo "Bootstrapping pkgdev via cargo from toasterson/forge"
+	cargo install --git https://github.com/toasterson/forge pkgdev
```

Comment:
Could you please publish forge at crates.io and then we should package it regularly.  This `cargo install` looks like a nasty hack we should avoid.

Thank you.